parlai/core/params.py

```diff
@@ -362,17 +376,36 @@ def add_parlai_args(self, args=None):
             help='Whether the agent should preprocess the data while building'
                  'the pytorch data')
         pytorch.add_argument(
-            '--batch-sort-cache', type=str,
-            choices=['pop', 'index', 'none'], default='none',
-            help='Whether to have batches of similarly sized episodes, and how'
-            'to build up the cache')
+            '-pybsrt', '--pytorch-teacher-batch-sort',
+            type='bool', default=False,
+            help='Whether to construct batches of similarly sized episodes'
+            'when using the PytorchDataTeacher (either via specifying `-pyt`'
+            'or `-pytd`')
```

Comment:
maybe we should actually turn it on by default for pytorch teacher?

before we disabled it just because of how it "corrupted" the data with newline-concat, but this version doesn't have that problem